play/client/src/test/scala/endpoints4s/play/client/PlayClientEndpointsUrlEncodingTest.scala

```diff
@@ -0,0 +1,17 @@
+package endpoints4s.play.client
+
+import endpoints4s.algebra
+import play.api.test.WsTestClient
+
+import scala.concurrent.ExecutionContext
+
+class PlayClientEndpointsUrlEncodingTest extends algebra.client.UrlEncodingTestSuite[TestClient] {
+
+  import ExecutionContext.Implicits.global
+
+  val wsClient = new WsTestClient.InternalWSClient("http", 8080)
```

Comment:
Should we also `close()` the client afterAll the tests have run?

Reply:
There are a few errors in CI that seem to indicate something being open when it should not be, I will try this out.